docs/usage_stats_collection.md

```diff
@@ -0,0 +1,54 @@
+# Usage Stats Collection
```

Comment:
Rename mentions of "usage" to "performance". E.g. the title should be "Performance stats". Also update this in the body.